source/docs/software/getting-started/driver-station-best-practices.rst

```diff
@@ -0,0 +1,72 @@
+Driver Station Best Practices
+=============================
+
+This document was created by Steve Peterson, with contributions from Juan Chong, James Cole-Henry, Rick Kosbab, Greg McKaskle, Chris Picone, Chris Roadfeldt, Joe Ross, and Ryan Sjostrand. The original post and followup posts can be found here: https://www.chiefdelphi.com/media/papers/3452 and a mirror of the document can be found `here. <https://media.screensteps.com/attachment_assets/assets/001/433/381/original/Driver_Station_Best_Practices_--_March_14_2018.pdf>`_
```

Comment:
Replace the link to the screensteps. Just put a mirror of the copy in the "assets" directory relative to the location of this document.